### ISSUE TITLE
Incorrect chinese characters received from loklak suggest.json API

### ISSUE THREAD

**Issue by Achint08** [loklak/loklak_server#1360]
- Issue type: Bug report

### Short description

Some incorrect Chinese characters are received from the server side by suggest.json API. Please look into this.

![screen shot 2017-07-25 at 3 08 52 pm](https://user-images.githubusercontent.com/15813932/28565958-6257a6a4-714b-11e7-9136-0c3c7df3d9ba.png)

**Comment by vibhcool** [comment]
@Achint08 , how did you reproduce this

**Comment by Achint08** [comment]
Ok. My system has a support for Chinese characters. Secondly, Please look into the issue https://github.com/fossasia/loklak_search/issues/491. Since loklak search just renders what is received from the server. I think it must be checked once here at server side for the issue. Please correct me If I am wrong somewhere.

**Comment by vibhcool** [comment]
yes, just checked https://github.com/fossasia/loklak_search/issues/491

**Comment by singhpratyush** [comment]
@Achint08: Could you please provide an example query?
